Resources/doc/Metric-types.rst

```diff
@@ -0,0 +1,16 @@
+Metric types tab
+================
+
+This tab is defining Metric type list.
+
+Currently, options are:
+
+- ``Area``
+- ``Binary``
+- ``Frequency``
+- ``Length``
+- ``Power``
+- ``Speed`` 
+- ``Temperature``
+- ``Volume`` 
+- ``Weight``
```

Comment:
if there are missing in this list, they are missing in the init.xslx file too. I should add'em inside the file as well